### DISCUSSION TITLE
How to add a connection to DBeaver with a SSH Jumphost?

### DISCUSSION THREAD

**Discussion by scobby** [dbeaver/dbeaver#8505] [Q&A] [answered]
#### System information: 
- Operating system : Mac OS
- DBeaver version: 7.0.2

#### Connection specification:
- no connection specification

#### Describe your question:
How is it possible to add a connection via CLI (Bash) with a SSH Tunnel?

**Comment by uslss** [comment, marked as answer]
Hi @scobby ,
you could do it using Shell Commands in connection properties
![image](https://user-images.githubusercontent.com/31996417/88382133-81922c00-cdb0-11ea-881e-ea28394334d2.png)
